oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -271,7 +298,7 @@ public void afterBulk(long executionId, BulkRequest bulkRequest, BulkResponse bu
 
         @Override
         public void afterBulk(long executionId, BulkRequest bulkRequest, Throwable throwable) {
-            LOG.error("Bulk with id {} threw an error", executionId, throwable);
+            LOG.error("ElasticIndex Update Bulk Failure : Bulk with id {} threw an error", executionId, throwable);
```

Comment:
if we are here the entire bulk has failed (eg: cannot connect to elastic). Should we track all the docs in the `BulkRequest`?

Reply:
@fabriziofortino  - I had tried that, but BulkItemRequest doesn't have any way to get the doc ID. 
BulkItemRequest.id() returns int, so I assume that is the id assigned to that item. 

We can probably create some alert in splunk over such logs, since this would suggest Elastic server is down. wdyt ?

Reply:
Makes sense.